tests/Symfony/Upgrade/Test/Fixer/FormTypeNamesFixerTest.php

```diff
@@ -14,8 +14,10 @@ public function testFix($expected, $input, $file)
 
     public function provideExamples()
     {
+        $outputFile = PHP_VERSION_ID > 50500 ? 'case1-output-php55+.php' : 'case1-output-php54.php';
+
         return [
-            $this->prepareTestCase('case1-output.php', 'case1-input.php'),
```

Comment:
Run both `$this->prepareTestCase('case1-output.php', 'case1-input.php'),` and `$this->prepareTestCase('case2-output.php', 'case2-input.php'),` test cases no matter what is PHP version.